dm/dm/ctl/master/start_stop_validation.go

```diff
@@ -39,7 +39,7 @@ func NewStartValidationCmd() *cobra.Command {
 		RunE:  startValidation,
 	}
 	cmd.Flags().Bool("all-task", false, "whether applied to all tasks")
-	cmd.Flags().String("mode", "", "specify the mode of validation: full, fast")
+	cmd.Flags().String("mode", "", "specify the mode of validation: full (default), fast")
```

Comment:
```suggestion
	cmd.Flags().String("mode", "full", "specify the mode of validation: full (default), fast")
```

why the default value is not set here 🤔

Reply:
It's set in the server (I didn't notice this either until I found the implicit logic in the server code😂)

https://github.com/pingcap/tiflow/blob/83a7cd9a1879e73e4077511f209b8c247da3c76e/dm/dm/master/server.go#L2748-L2751

But maybe the tips should be revised too, because the `full` mode is by default only when the user has never enabled the validator while, on the contrary, the `mode` cannot be reset by `validation start --mode=xxx`, in the case of which `mode` is ignored.

Reply:
seems we can set default as @lance6716 suggested, since we already checked whether there's explicit mode or not.

Reply:
Will fix it.

Reply:
d28f43f